### PR TITLE
update mem limit of kube-rbac-proxy container in pod of node-exporter to 60M 

### LINKED ISSUE #91: node-exporter OOM
**What happened?**
node-exporter oom many times,should we increase the limit?
https://github.com/coreos/kube-prometheus/blob/master/jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet#L106

**Did you expect to see some different?**

**How to reproduce it (as minimally and precisely as possible)**:

**Environment**

* Prometheus Operator version:

    `Insert image tag or Git SHA here`
    

* Kubernetes version information:

    `kubectl version`
    

* Kubernetes cluster kind:

    insert how you created your cluster: kops, bootkube, tectonic-installer, etc.

* Manifests:

```
insert manifests relevant to the issue
```

* Prometheus Operator Logs:

```
Insert Prometheus Operator logs relevant to the issue here
```

* Prometheus Logs:

```
Insert Prometheus logs relevant to the issue here
```

**Anything else we need to know?**:

### PR DESCRIPTION
**Pull request by zgfh**
update mem limit of kube-rbac-proxy container in pod of node-exporter to 60M 
fix #91

### CODE REVIEW

**Comment by zgfh** [comment]
@brancz build is not work,fixed it too
image

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ all: generate fmt test
 .PHONY: generate-in-docker
 generate-in-docker:
 	@echo ">> Compiling assets and generating Kubernetes manifests"
-	$(CONTAINER_CMD) $(MAKE) $(MFLAGS) generate
+	$(CONTAINER_CMD) make $(MFLAGS) generate
```
Comment (brancz) [inline Makefile]: I’m not sure why that change was necessary for you. Can you revert this part? https://www.gnu.org/software/make/manual/html_node/Recursion.html

**Comment by zgfh** [comment]
please have a look the error in the picture ， when build with mac，it replace ${MAKE} with host path of make，but the path is not existed in container 

--------------原始邮件--------------
发件人："Frederic Branczyk "<notifications@github.com>;
发送时间：2019年5月18日(星期六) 中午11:27
收件人："coreos/kube-prometheus" <kube-prometheus@noreply.github.com>;
抄送："Alan "<zzg@daozzg.com>;"Author "<author@noreply.github.com>;
主题：Re: [coreos/kube-prometheus] update mem limit of kube-rbac-proxy container in pod of node-exporter to 60M  (#103)
-----------------------------------

 
@brancz commented on this pull request.
 
Just one small nit. Otherwise lgtm

 
In Makefile:
 > @@ -14,7 +14,7 @@ all: generate fmt test  .PHONY: generate-in-docker  generate-in-docker:  	@echo ">> Compiling assets and generating Kubernetes manifests" -	$(CONTAINER_CMD) $(MAKE) $(MFLAGS) generate +	$(CONTAINER_CMD) make $(MFLAGS) generate  
I’m not sure why that change was necessary for you. Can you revert this part? https://www.gnu.org/software/make/manual/html_node/Recursion.html

**Comment by brancz** [comment]
Ah sorry lgtm :+1:

**Pull request merged**